### PR TITLE
Second task

### PR DESCRIPTION
**Pull request by an2onsol**
https://projecteuler.net/problem=2

### CODE REVIEW

File: second_problem_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+describe '#even_fibonacci' do
+  it 'should return right values' do
+    expect(even_fibonacci(2)).to eq 2
+    expect(even_fibonacci(1)).to eq 0
+    expect(even_fibonacci(10)).to eq 10
+    expect(even_fibonacci(50)).to eq 44
+    expect(even_fibonacci(4000000)).to eq 4613732
+  end
+end
```
Comment (AndreyAnischik) [inline second_problem_spec.rb:12]: space

File: second_problem.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+def even_fibonacci(arg)
+  first = 1
+  second = 2
+  sum = 0
+
+  while second <= arg
+    if second % 2 == 0
```
Comment (AndreyAnischik) [inline second_problem.rb:7]: use `even?`

File: second_problem.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      sum += second
+    end
+
+    first, second = second, first + second
+  end
+  return sum
```
Comment (AndreyAnischik) [inline second_problem.rb:13]: no need to use return

**Review by AndreyAnischik — APPROVED** [review]

File: second_problem.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+def even_fibonacci(arg)
+  first = 1
+  second = 2
+  sum = 0
+
+  while second <= arg
+    if second.even? == 0
```
Comment (bronium) [inline second_problem.rb:7]: `even?` returns boolean, so comparing it with 0 is redundant

**Review by bronium — APPROVED** [review]

**Pull request closed without merging**